codebuild/bin/install_awslc.sh

```diff
@@ -43,7 +43,8 @@ cd "$BUILD_DIR"
 echo "Checking out tag=$AWSLC_VERSION"
 # --branch can also take tags and detaches the HEAD at that commit in the resulting repository
 # --depth 1 Create a shallow clone with a history truncated to 1 commit
-git clone https://github.com/awslabs/aws-lc.git --branch "$AWSLC_VERSION" --depth 1
+# If the curl above is throttled, fall back to a known version.
+git clone https://github.com/awslabs/aws-lc.git --branch "${AWSLC_VERSION:-v1.48.4}" --depth 1
```

Comment:
My concern with this is that there's nothing forcing this pinned version to be bumped. For example, we may add a new test that tests something in a recent AWS-LC release. To do this, we'll bump the AWS-LC version in the docker images by rebuilding them - no install script changes should be needed now. But if this pin remains fixed, we may assume that the new test will be tested against the new AWS-LC release, when it might not be due to falling back to the pin.

It seems better to always pin this to ensure we're testing with the AWS-LC release that we think we are. Alternatively, I wonder if we could just fallback to main rather than a pinned version? main would be more similar to the latest release and would always be newer than what we assume, rather than older.

Reply:
Falling back to main sgtm.